services/ui-src/src/components/export/ExportedModalDrawerReportSection.tsx

```diff
@@ -17,21 +17,35 @@ export const ExportedModalDrawerReportSection = ({
   const sectionHeading = verbiage?.intro.subsection || name;
   const entityCount = report?.fieldData?.[entityType]?.length;
 
-  let emptyEntityMessage = "";
+  let emptyEntityMessage;
   switch (entityType) {
     case ModalDrawerEntityTypes.ACCESS_MEASURES:
-      emptyEntityMessage = `No access and adequacy measures have been entered for this program report.`;
+      emptyEntityMessage = " 0 -- No access measures entered";
       break;
     case ModalDrawerEntityTypes.SANCTIONS:
-      emptyEntityMessage = `No plan-level sanctions or corrective actions have been entered for this program report.`;
+      emptyEntityMessage = " 0 -- No sanctions entered";
       break;
     case ModalDrawerEntityTypes.QUALITY_MEASURES: {
-      emptyEntityMessage = `No quality measures and plan-level quality measure results have been entered for this program report.`;
+      emptyEntityMessage = " 0 -- No quality measures entered";
       break;
     }
     default:
       break;
   }
+
+  const headerEntityCount = (
+    <Heading as="h3" sx={sx.dashboardTitle} data-testid="headerCount">
+      {verbiage.dashboardTitle}
+      {entityCount ? (
+        ` ${entityCount}`
+      ) : (
+        <Text as="span" sx={sx.notAnswered} data-testid="entityMessage">
+          {emptyEntityMessage}
+        </Text>
+      )}
+    </Heading>
```

Comment:
Thought, continued from above: Then we could simplify this like:

```suggestion
    <Heading as="h3" sx={sx.dashboardTitle} data-testid="headerCount">
      {verbiage.dashboardTitle} ${entityCount} - 
      {!entityCount && (
        <Text as="span" sx={sx.notAnswered} data-testid="entityMessage">
          {emptyEntityMessage}
        </Text>
      )}
    </Heading>
```

`entityCount` always shows because the count number should always show, and the additional empty message only shows if there are no entities.

Reply:
One thing here is that the `0` has to be red as well, but I do like this idea.

Reply:
this change gives the desired effect:
```

  const entityCount = report?.fieldData?.[entityType]?.length;

  let emptyEntityMessage;
  switch (entityType) {
    case ModalDrawerEntityTypes.ACCESS_MEASURES:
      emptyEntityMessage = "No access measures entered";
      break;
    case ModalDrawerEntityTypes.SANCTIONS:
      emptyEntityMessage = "No sanctions entered";
      break;
    case ModalDrawerEntityTypes.QUALITY_MEASURES: {
      emptyEntityMessage = "No quality measures entered";
      break;
    }
    default:
      break;
  }

  const headerEntityCount = (
    <Heading as="h3" sx={sx.dashboardTitle} data-testid="headerCount">
      {verbiage.dashboardTitle} {entityCount} 
      {!entityCount && (
        <Text as="span" sx={sx.notAnswered} data-testid="entityMessage">
          0 - {emptyEntityMessage}
        </Text>
      )}
    </Heading>
  );
```